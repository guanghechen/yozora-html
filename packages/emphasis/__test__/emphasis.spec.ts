import type { Emphasis, Text } from '@yozora/ast'
import { EmphasisType, TextType } from '@yozora/ast'
import { renderChildren } from '../../../jest.setup'
import renderEmphasis from '../src'

describe('snapshot', function () {
  it('basic', function () {
    const emphasis: Emphasis = {
      type: EmphasisType,
      children: [
        {
          type: TextType,
          value: 'Hello',
        } as Text,
        {
          type: TextType,
          value: 'World',
        } as Text,
      ],
    }

    expect(renderEmphasis(emphasis, renderChildren)).toMatchSnapshot()
  })
})
