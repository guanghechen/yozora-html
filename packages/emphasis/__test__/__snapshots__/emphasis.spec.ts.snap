// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot basic 1`] = `"<em class=\\"yozora-emphasis\\"><span class=\\"yozora-text\\">Hello</span><span class=\\"yozora-text\\">World</span></em>"`;
